### PR TITLE
Merge gradle into develop.

### PR DESCRIPTION
**Pull request by SerbianSensation**
Final pull request for the sprint (hopefully). Made changes to CommandLine.java and Classes.java to remove bugs and print more specific error messages. Also deleted unnecessary files.

### CODE REVIEW

**Comment by HomelessSkittle** [comment]
I think it's good. Did a quick test of my own and it worked.

**Review by HomelessSkittle — APPROVED** [review]
Looks good to me! Did a quick test of my own and it worked

File: src/main/java/mike/cli/CommandLine.java [hunk 1]
```diff
@@ -94,20 +94,36 @@ public static void commandInterface() {
...
+							System.out.println("\nCreate field failed. Make sure the field doesn't already exist and the class name does exists.\n");
 						}
 					} else if ( commands[1].equals("method")) {
-						if(commands.length != 4 || !userClasses.createMethod(commands[2], commands[3])) {
+						if(commands.length != 4) {
 							System.out.println(errorMessage + commandUsage[4] + "\n");
+							break;
+						}
+						if(!userClasses.createMethod(commands[2], commands[3])){
+							System.out.println("\nCreate method failed. Make sure the method doesn't already exist and the class name does exists.\n");
```
Comment (HomelessSkittle) [inline src/main/java/mike/cli/CommandLine.java:118]: Just a typo - delete the "s" at the end of exists

"Make sure the method doesn't already exist, and the class name does exist**s**"

File: src/main/java/mike/cli/CommandLine.java [hunk 2]
```diff
@@ -94,20 +94,36 @@ public static void commandInterface() {
...
+							System.out.println("\nCreate class failed. Make sure the class name doesn't already exist.\n");
 						}
 					} else if ( commands[1].equals("field")) {
-						if(commands.length != 4 || !userClasses.createField(commands[2], commands[3])) {
+						if(commands.length != 4) {
 							System.out.println(errorMessage + commandUsage[3] + "\n");
+							break;
+						}
+						if(!userClasses.createField(commands[2], commands[3])){
+							System.out.println("\nCreate field failed. Make sure the field doesn't already exist and the class name does exists.\n");
```
Comment (HomelessSkittle) [inline src/main/java/mike/cli/CommandLine.java:110]: Just a typo - delete the "s" at the end of exists

"Make sure the method doesn't already exist, and the class name does exist**s**"

**Review by HomelessSkittle — COMMENTED** [review]
Two typos but that's it.

**Review by phila413 — APPROVED** [review]
Looks good to me

**Review by HomelessSkittle — APPROVED** [review]
Looks like a quality UML to me!

**Pull request merged**